soc/arm/nxp_imx/rt/arm_mpu_mem_cfg.h

```diff
@@ -22,6 +22,8 @@
 #define REGION_FLASH_SIZE REGION_1M
 #elif CONFIG_FLASH_SIZE == 2048
 #define REGION_FLASH_SIZE REGION_2M
+#elif CONFIG_FLASH_SIZE == 4096
+#define REGION_FLASH_SIZE REGION_4M
```

Comment:
Move this change to the previous commit that added support for the internal flash.

Reply:
ok